bin/antigen.zsh

```diff
@@ -364,6 +375,8 @@ antigen () {
       branch="${url#*|}"
     fi
     install_or_update=true
+    # Update remote if needed.
+    -antigen-update-remote $clone_dir $url
```

Comment:
Makes sense to add a configuration variable for the preferred prezto fork.

Are you suggesting here a second variable, which determines whether remotes should be updated for existing clones? If a user decides to switch forks and adjusts configuration appropriately, it doesn't make sense for them to set an additional variable to get antigen to actually install the fork.